.vib/redmine/goss/goss.yaml

```diff
@@ -1,40 +1,14 @@
-command:
-  ruby-version-test:
-    exec: ruby -v
-    exit-status: 0
-    stdout:
-      - ruby
-    stderr: [ ]
-    timeout: 20000  
 file:
```

Comment:
You missed testing the security context-related params and their related `id` test. Additionally, would it be possible to test the `containerPorts.http` param?

Reply:
The `id` test was skipped intentionally because the securityContext feature for this chart is disabled by default. On the other hand, the test for `containerPorts.http` param was not included as the chart has trouble changing the default value.